src/app/components/ui/ukhsa/MiniMap/MiniMapCard.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import { render, screen } from '@/config/test-utils'
+
+import { MiniMapCard } from './MiniMapCard'
+
+//
+jest.mock('./MiniMap', () => ({
+  MiniMap: jest.fn(() => <div>MiniMap Mock Component</div>),
+}))
+
+const renderMinimapComponent = (type: 'heat' | 'cold') => {
+  const title = `${type} Health Alert`
+  const subTitle = `${type} temperatures expected`
+  const alertType = type
+
+  const { container } = render(<MiniMapCard title={title} subTitle={subTitle} alertType={alertType} />)
+  return {
+    container,
+  }
+}
+
+describe('MiniMapCard Component', () => {
+  it('renders the heading with the correct title text', () => {
+    renderMinimapComponent('heat')
+    const heading = screen.getByRole('heading', { level: 3 })
+    expect(heading).toHaveTextContent('heat Health Alert')
+  })
+
+  it('renders the subtitle correctly', () => {
+    renderMinimapComponent('cold')
+    const subtitle = screen.getByText('cold temperatures expected')
+    expect(subtitle).toBeInTheDocument()
```

Comment:
Same comment as above: 

```suggestion
    expect(screen.getByText('cold temperatures expected')).toBeInTheDocument()
```